pom.xml

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apHashiCorpSecretRepositoryache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.wso2.carbon</groupId>
+    <artifactId>org.wso2.carbon.securevault.hashicorp</artifactId>
+    <version>1.0</version>
+    <packaging>bundle</packaging>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Export-Package>
+                            org.wso2.carbon.securevault.hashicorp.common;version="${project.version}",
+                            org.wso2.carbon.securevault.hashicorp.config;version="${project.version}",
+                            org.wso2.carbon.securevault.hashicorp.exception;version="${project.version}",
+                            org.wso2.carbon.securevault.hashicorp.repository;version="${project.version}"
+                        </Export-Package>
+                        <Import-Package>
+                            com.bettercloud.vault.*,
```

Comment:
Import specific package, instead of importing all the packages.

Reply:
Resolved in #6